Cargo.toml

```diff
@@ -22,3 +22,5 @@ rusty-fork = { git = "https://github.com/daxpedda/rusty-fork", branch = "proc-ma
 [features]
 nightly = []
 test = []
+custom-transport = ["sys/custom-transport"]
+copy-handler = []
```

Comment:
I think we can just make this the default behaviour, no need to add another feature, what do you say?